### PR TITLE
hotdog: set images to save

### PR DESCRIPTION
**Pull request by ShevT**

### CODE REVIEW

File: build_targets [hunk 1]
```diff
@@ -7,7 +7,7 @@ beryllium userdebug yes [recovery.img]
 #courbet user yes yes
 #dumpling user yes no
 guacamole userdebug yes [boot.img]
-#hotdog userdebug yes yes
+hotdog userdebug yes [recovery.img, dtbo.img, vbmeta.img]
```
Comment (gwolf2u) [inline build_targets:10]: shouldn't be boot instead of recovery?

Comment (ShevT) [reply build_targets:10]: No. We have a separate partition - recovery

Comment (gwolf2u) [reply build_targets:10]: guacamole uses boot, reason I asked

**Review by gwolf2u — APPROVED** [review]

File: build_targets (hunk 1 above)
Comment (ShevT) [reply build_targets:10]: Also, LineageOS recommended dtbo and vbmeta - https://wiki.lineageos.org/devices/hotdog/install/#flashing-additional-partitions

Comment (ShevT) [reply build_targets:10]: > guacamole uses boot, reason I asked

Yes, I remember. We once discussed this in the Telegram

**Pull request merged**